composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/BottomNavigationBar.kt

```diff
@@ -48,6 +48,7 @@ fun BottomNavigationBar(
         modifier = customMinHeightModifier,
     ) {
         MAIN_NAVIGATION_SCREENS.forEach { screen ->
+            val screen = screen as Screen
```

Comment:
Was lint complaining or something?

Reply:
iOS was complaining that screen is of type `Any`